test/test-korebuild.ps1

```diff
@@ -0,0 +1,11 @@
+$ErrorActionPreference ='Stop'
```

Comment:
Maybe add a README.md to this folder with the basic description of what this is for? (Essentially your PR description?)